scripts/ait_bootstrap.sh

```diff
@@ -88,6 +88,11 @@ cd $PROJECT_HOME/AIT-GUI/
 git checkout 2.3.1
 pip install .
 
+git clone https://github.com/NASA-AMMOS/AIT-DSN.git $PROJECT_HOME/AIT-DSN
+cd $PROJECT_HOME/AIT-DSN
+git checkout 2.0.0
+pip install .
+
```

Comment:
Let's go ahead and add a `cd $PROJECT_HOME` here once we're done with the git operations